core/layout_api.php

```diff
@@ -936,7 +929,6 @@ function layout_breadcrumbs() {
 	echo '<div id="breadcrumbs" class="breadcrumbs">' , "\n";
 
 	# Login information
-	echo '<ul class="breadcrumb">' , "\n";
```

Comment:
Breadcrumbs traditionally represented by <ul> element. We are not show good breadcrumbs yet, but I think we should do that instead removing the list.